metrotranscom/doorway
# Jurisdictional Refactor Feature Merge

# Pull Request Template

## Issue Overview

This PR addresses #319 


## Description

This PR merges the jurisdictional refactor feature branch into main. it also includes 2 minor testing fixes that came to light in this PR.

## How Can This Be Tested/Reviewed?

The best way to test this is to mimic how this will work when releasing this to prod:

1. **Before you pull down this change** - in the main branch do a reseed
2. Pull this change down and in the backend/core directory run `yarn db:migration:run`
3. Migration should run correctly and should be able to start up the app
4. The partner site should show the following changes:
  - When adding a new partner user all of the new jurisdictions should exist
  - When adding a new listing the jurisdiction dropdown should appear with all of the new jurisdictions as options
  - AMI charts should be tied to appropriate jurisdictions

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
